lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-network_manager-cloud_subnet.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_NetworkManager_CloudSubnet < MiqAeServiceCloudSubnet
+    expose :delete_cloud_subnet_queue
```

Comment:
@andyvesel The backend model method requires a `userid` to be passed [app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb#L49](https://github.com/ManageIQ/manageiq/blob/5ce005bb845c841ce2af65a19e985253f82a63d2/app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb#L49)

I assume the intent here is that the user would not be expect to pass a valid `userid` value, correct?  

Can you provide some insight into what you would expect.  Should the `userid` always be from the automate context or is there a valid reason to allow them to pass this parameter from the automate method?

cc @mkanoor